src/Components/ProgressBar2.js

```diff
@@ -0,0 +1,94 @@
+import React, { useRef, useEffect } from 'react';
+import { StyleSheet, Animated, View, TouchableOpacity } from 'react-native';
+
+/** Progress bar displayed with an animation which:
```

Comment:
This comment is a bit confusing since you use it to display the progression within the training. Maybe we should remove it?